site/releases/2025/2025-apr-24/release-notes.qmd

```diff
@@ -0,0 +1,284 @@
+--->
+#### Ability to manage model stakeholder types
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+We've introduced the ability to define custom model stakeholder types, allowing you to control granular permissions on each model in your inventory:
+
+:::
+
+::: {.w-40-ns .tc}
+[Manage model stakeholder types](/guide/configuration/manage-model-stakeholder-types.qmd){.button .button-green target="_blank"}
+
+:::
+
+::::
+
+- Model stakeholders determine specific responsibilities and access levels for model review and approval processes for each model in your model inventory, such as read or edit access to model inventory fields.
+- Each model stakeholder also belongs to user groups[^1] which determine which models they can see, and have user roles[^2] with attached role permissions[^3] which define the level of access they have to overarching {{< var validmind.platform >}} features.
+
+::: {.callout}
+Model stakeholders are now also available as a field defined in conditional arguments within model workflow steps.[^4]
+
+:::
+
+### Documentation
+
+<!---
+PR #640: Switch Python API reference to Quarto
+URL: https://github.com/validmind/documentation/pull/640
+Labels: documentation, highlight
+--->
+#### Improved {{< var vm.api >}} reference
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+We've completely redesigned our {{< var validmind.api >}} reference using the same tools we use to produce the rest of our product documentation, allowing us to more easily keep this information up to date and ensure its accuracy.
+
+:::
+
+::: {.w-40-ns .tc}
+[{{< var validmind.api >}} reference](/validmind/validmind.qmd){.button .button-green target="_blank"}
+
+:::
+
+::::
+
+- Now featuring more intuitive navigation wrapped in a familiar sidebar, enhanced code signature styling, and integration with our main docs site search, these improvements aim to empower users to maximize the potential of the {{< var validmind.developer >}}.
+- The updated reference structure mirrors the Python package layout, ensuring backward compatibility with our old Python API reference while providing a more reader-friendly experience.
+
+::: {.column-margin}
+:::{#python-api}
+:::
+
+:::
+
+:::
+
+## Enhancements
+
+### {{< var validmind.developer >}} (v2.8.13)
+
+<!---
+PR #337: [SC 8805] support config option for logging test results
+URL: https://github.com/validmind/validmind-library/pull/337
+Labels: documentation, enhancement
+--->
+#### Support for display configuration of test-driven blocks
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-80-ns}
+You can now customize the display of test results when inserted as test-driven blocks in the {{< var validmind.platform >}} when logging results via the {{< var validmind.developer >}} with `result.log()`.
+
+:::
+
+::: {.w-20-ns .tc}
+[result.log()](/validmind/validmind/vm_models.qmd#log){.button target="_blank"}
+
+:::
+
+::::
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-30-ns}
```

Comment:
```suggestion
::: {.w-30-ns .pr1}
```

Just adding a bit of padding here and in the right column, see next comment. Tested locally, works without causing text wrapping.